cmd/helm/root_unix.go

```diff
@@ -27,7 +27,7 @@ import (
 func checkPerms() {
 	// This function MUST NOT FAIL, as it is just a check for a common permissions problem.
 	// If for some reason the function hits a stopping condition, it may panic. But only if
-	// we can be sure that it is panicing because Helm cannot proceed.
+	// we can be sure that it is panicking because Helm cannot proceed.
```

Comment:
Yes, I know some people think the former is a thing. Neither [books](https://books.google.com/ngrams/graph?content=panicking%2Cpanicing&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cpanicking%3B%2Cc0%3B.t1%3B%2Cpanicing%3B%2Cc0) nor the [Internet](https://books.google.com/ngrams/graph?content=panicking%2Cpanicing&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cpanicking%3B%2Cc0%3B.t1%3B%2Cpanicing%3B%2Cc0) agree.